netmap/netmap_contract.go

```diff
@@ -35,10 +35,13 @@ const (
 	notaryDisabledKey = "notary"
 	innerRingKey      = "innerring"
 
-	snapshot0Key     = "snapshotCurrent"
-	snapshot1Key     = "snapshotPrevious"
-	snapshotEpoch    = "snapshotEpoch"
-	snapshotBlockKey = "snapshotBlock"
+	// SnapshotCount contains the number of previous snapshots stored by this contract.
+	// Must be less than 255.
+	SnapshotCount        = 10
```

Comment:
- may it be useful to be able to change that value?
- may it be useful to be able to get that value from the contract?

Reply:
Both yes.